code/modules/antagonists/cult/cult.dm

```diff
@@ -295,7 +295,7 @@
 /proc/is_convertable_to_cult(mob/living/target, datum/team/cult/specific_cult)
 	if(!istype(target))
 		return FALSE
-	if(isnull(target.mind) || !GET_CLIENT(target))
+	if(isnull(target.mind))// || !GET_CLIENT(target))
```

Comment:
?

Reply:
No reason for it to exist was just annoying when trying to test stuff